src/components/FavoritesBtn.tsx

```diff
@@ -9,14 +9,14 @@ type FavoritesBtnProps = {
 };
 
 const FavoritesBtn: React.FC<FavoritesBtnProps> = ({ movieId }) => {
-  const [isFavorite, setIsFavorite] = useState(api.isIdInFavorites(movieId));
- 
+  const [isFavorite, setIsFavorite] = useState(!api.isIdInFavorites(movieId));
+
   const heartUnfilledIcon = '/images/favoriteBtn/heartFilled.svg';
   const heartFilledIcon = '/images/favoriteBtn/heartUnfilled.svg';
 
   const handleSwitchFavoriteState = (): void => {
     isFavorite ? api.setFavoritesId(movieId) : api.deleteFavoritsId(movieId);
-    setIsFavorite(api.isIdInFavorites(movieId));
+    setIsFavorite(!api.isIdInFavorites(movieId));
```

Comment:
a bit confusing, we need to switch isFavorite? why not setIsFavorite(!isFavorite); ?